examples/experimental/openvino/mobilenet_v2/main.py

```diff
@@ -0,0 +1,178 @@
+    ov.serialize(quantized_model, str(ir_qmodel_xml), str(ir_qmodel_bin))
+
+    # Step 6: Compare the accuracy of the original and quantized models.
+    print('Checking the accuracy of the original model:')
+    metric = validation_fn(ov_model, val_dataloader)
+    print(f'accuracy@top1: {metric}')
+
+    print('Checking the accuracy of the quantized model:')
+    metric = validation_fn(quantized_model, val_dataloader)
+    print(f'accuracy@top1: {metric}')
+
+    # Step 7: Compare Performance of the original and quantized models.
+    # benchmark_app -m mobilenet_v2_quantization/mobilenet_v2.xml -d CPU -api async
+    # benchmark_app -m mobilenet_v2_quantization/mobilenet_v2_quantized.xml -d CPU -api async
+
+
+def convert_torch_to_openvino(model: torch.nn.Module) -> ov.Model:
+    """
+    Converts PyTorch MobileNetV2 model to the OpenVINO IR format.
+    """
+    if not MODEL_DIR.exists():
+        os.makedirs(MODEL_DIR)
+
+    # Export PyTorch model to the ONNX format.
+    onnx_model_path = MODEL_DIR.joinpath('mobilenet_v2.onnx')
+    dummy_input = torch.randn(1, 3, 224, 224)
+    torch.onnx.export(model, dummy_input, onnx_model_path, verbose=False)
+
+    # Run Model Optimizer to convert ONNX model to OpenVINO IR.
+    mo_command = f'mo --framework onnx -m {onnx_model_path} --output_dir {MODEL_DIR}'
+    subprocess.call(mo_command, shell=True)
+
+    ir_model_xml = MODEL_DIR.joinpath('mobilenet_v2.xml')
+    ir_model_bin = MODEL_DIR.joinpath('mobilenet_v2.bin')
+    ov_model = ie.read_model(model=ir_model_xml, weights=ir_model_bin)
+
+    return ov_model
+
+
+def create_data_source() -> torch.utils.data.DataLoader:
+    """
+    Creates validation ImageNet data loader.
+    """
+    val_dir = DATASET_DIR.joinpath('val')
+    # Transformations were taken from [here](https://pytorch.org/hub/pytorch_vision_mobilenet_v2).
+    preprocess = torchvision.transforms.Compose([
+        torchvision.transforms.Resize(256),
+        torchvision.transforms.CenterCrop(224),
+        torchvision.transforms.ToTensor(),
+        torchvision.transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+    ])
+    val_dataset = torchvision.datasets.ImageFolder(val_dir, preprocess)
+    val_dataloader = torch.utils.data.DataLoader(val_dataset,
+                                                 batch_size=1,
+                                                 shuffle=False,
+                                                 num_workers=4)
+    return val_dataloader
+
+
+# The `validate()` method was taken as is from the pythorch repository.
+# You can find it [here](https://github.com/pytorch/examples/blob/main/imagenet/main.py).
+# Code regarding CUDA and training was removed.
+# Some code was changed and added.
+
+def validation_fn(model: ov.Model, data_loader: nncf.DataLoader) -> float:
+    compiled_model = ie.compile_model(model, device_name='CPU')
+    output_layer = next(iter(compiled_model.outputs))
```

Comment:
For me `output_layer = compiled_model.output(0)` is more readable

Reply:
Thanks for the comment! I fixed it.